ci/authn-k8s/test_gke_entrypoint.sh

```diff
@@ -128,45 +129,49 @@ function launchConjurMaster() {
     sed -e "s#{{ CONJUR_AUTHN_K8S_TEST_NAMESPACE }}#$CONJUR_AUTHN_K8S_TEST_NAMESPACE#g" |
     kubectl create -f -
 
-  conjur_pod=$(kubectl get pods -l app=conjur-authn-k8s -o=jsonpath='{.items[].metadata.name}')
+  conjur_pod=$(retrieve_pod conjur-authn-k8s)
 
-  wait_for_it 300 "kubectl describe po $conjur_pod | grep Status: | grep -q Running"
+  kubectl wait --for=condition=Ready pod/$conjur_pod --timeout=5m
 
   # wait for the 'conjurctl server' entrypoint to finish
-  kubectl exec $conjur_pod -- bash -c "while ! curl -sI localhost:80 > /dev/null; do sleep 1; done"
+  local wait_command="while ! curl --silent --head --fail localhost:80 > /dev/null; do sleep 1; done"
+  kubectl exec $conjur_pod -- bash -c "$wait_command"
   
   export API_KEY=$(kubectl exec $conjur_pod -- conjurctl account create cucumber | tail -n 1 | awk '{ print $NF }')
 }
 
 function copyNginxSSLCert() {
-  nginx_pod=$(kubectl get pods -l app=nginx-authn-k8s -o=jsonpath='{.items[].metadata.name}')
-  cucumber_pod=$(kubectl get pods -l app=cucumber-authn-k8s -o=jsonpath='{.items[].metadata.name}')
+  #Copy Nginx SSL Cert
```

Comment:
Can you remove these comments? They're just mirroring the name of the function so they're not really adding any value.